lib/lita/handlers/jira_issues.rb

```diff
@@ -8,15 +8,18 @@ class JiraIssues < Handler
       config :url, required: true, type: String
       config :username, required: true, type: String
       config :password, required: true, type: String
+      config :ignore, default: [], type: Array
 
       route /[a-zA-Z]+-\d+/, :jira_message, help: {
         "KEY-123" => "Replies with information about the given JIRA key"
       }
 
       def jira_message(response)
-        @jira ||= JiraGateway.new(http, config)
-        Set.new(response.matches).each do | key |
-          handle_key(response, key)
+        return if config.ignore.include?(response.user.name)
```

Comment:
Sorry, that's an inline if statement, so the lines below should be at the same indentation (and there's an extra end). It should look something like:

```ruby
def jira_message(response)
  return if config.ignore.include?(response.user.name)
  @jira ||= JiraGateway.new(http, config)
  Set.new(response.matches).each do | key |
    handle_key(response, key)
  end
end
```